src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -0,0 +1,212 @@
+import classNames from 'classnames';
+import { useContext, useState } from 'react';
+import * as commentAPI from '../../api/commentAPI';
+import { PostContext } from '../../context/PostContext';
+import { Comment } from '../../types/Comment';
+
+export const NewCommentForm: React.FC = () => {
+  const [name, setName] = useState('');
+  const [nameError, setNameError] = useState(false);
+  const [email, setEmail] = useState('');
+  const [emailError, setEmailError] = useState(false);
+  const [text, setText] = useState('');
+  const [textError, setTextError] = useState(false);
```

Comment:
could you check how you can optimize your form? what if it's needed to add 3 more fields?

Reply:
it's not blocker, you can skip this comment, just try to thinking about it